sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
   - `ServiceBusReceiver`: `receive_deferred_messages` and `peek_messages`
   - `ServiceBusSession`: `get_state`, `set_state` and `renew_lock`
   - `ReceivedMessage`: `renew_lock`
+* Added a `ServiceBusConnectionStringParser` which parses a connection string into individual entities.
```

Comment:
```suggestion
* Added a `ServiceBusConnectionStringParser` which parses a connection string into a properties bag containing its component parts
```
nitpicky clarity. (If we change to a different parsing strategy should be able to just change the class name accordingly)